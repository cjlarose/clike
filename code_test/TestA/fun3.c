

// 4 args

void printint(int);
void printchar(char);

int f(x, y, z, c)
   int x, y, z;
   char c;
{
  return x + y * z + c;
}


void main()
{
  int x, y;

  x = 17;
  y = f(1, x, x*2, 100);

  printint(y);
}

