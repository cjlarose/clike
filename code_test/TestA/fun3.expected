Loaded: /usr/local/share/spim/exceptions.s
679